serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,51 @@
+= Migrate Your Postgresql database with Kogito upgrade
+
+Kogito provides out of the box way to manage your database schema changes with each upgrade. Kogito uses flyway to achieve the same functionality.
```

Comment:
According to the Quarkus and Flyway sites, it should be written with a capital first letter.
```suggestion
Kogito provides out of the box way to manage your database schema changes with each upgrade. Kogito uses Flyway to achieve the same functionality.
```